react-basic-practice/src/components/Modal/index.tsx

```diff
@@ -9,23 +9,34 @@ import {
 } from "@chakra-ui/react";
 import Button from "../Button";
 import Form from "../Form";
+import { ICustomer } from "@types";
 
 interface ModalProps {
   isOpen: boolean;
   onClose: () => void;
   headerText?: string;
+  viewData?: ICustomer | null;
```

Comment:
why we need `null` type here? Pls consider remove it. Optional is enough

Reply:
i done